client/src/components/Overview/PrimaryImage.jsx

```diff
@@ -0,0 +1,161 @@
+    $(e.target).addClass('active-thumb');
+    setActiveThumb(newInd);
+    setCurrentPic(newInd);
+    $('.carousel').carousel(newInd);
+    $('.carousel-control-next').removeClass('hidden');
+    $('.carousel-control-prev').removeClass('hidden');
+    if (newInd === 0) {
+      $('.carousel-control-prev').addClass('hidden');
+    } else if (newInd === picCount - 1) {
+      $('.carousel-control-next').addClass('hidden');
+    }
+  };
+
+  const picUpdate = (button, totalPics, index) => {
+    if (button === 'next') {
+      if (index === totalPics - 1) {
+        $('.carousel-control-next').addClass('hidden');
+      }
+      $('.carousel-control-prev').removeClass('hidden');
+      if (index > currentThumbs[1]) {
+        slideDown();
+      }
+    }
+    if (button === 'prev') {
+      if (index === 0) {
+        $('.carousel-control-prev').addClass('hidden');
+      }
+      $('.carousel-control-next').removeClass('hidden');
+      if (index < currentThumbs[0]) {
+        slideUp();
+      }
+    }
+    setActiveThumb(index);
+    setCurrentPic(index);
+  };
+
+  const nextStyle = () => {
+    setStyle(style + 1);
+  };
+
+
+
+  const nextClick = () => {
+    $('.active-thumb').removeClass('active-thumb');
+    $('img').find(`[data-index='${currentPic + 1}']`).addClass('active-thumb');
+    picUpdate('next', picCount, Number(currentPic + 1));
+  };
+
+  const prevClick = () => {
+    $('.active-thumb').removeClass('active-thumb');
+    $('img').find(`[data-index='${currentPic - 1}']`).addClass('active-thumb');
+    picUpdate('prev', picCount, Number(currentPic - 1));
+  };
+
+  const imageExpand = () => {
+    console.log('in expand');
+    if (imageExpanded) {
+      $('.main-pic').removeClass('col-12').addClass('col-8');
+      setImageExpanded(false);
+      setTimeout(() => {
+        $('.main-details').removeClass('hidden').addClass('col-4');
+      }, 1000);
+    } else {
+      $('.main-pic').removeClass('col-8').addClass('col-12');
+      $('.main-details').removeClass('col-4').addClass('hidden');
+      setImageExpanded(true);
+    }
+  };
+
+
+
+  return (
+    <div id="main-image">
+      {/* <button onClick={ nextStyle }>nextStyle</button> */}
+      <p className="up-button thumbNav" onClick={ slideUp }>&#8963;</p>
+      <p className="down-button thumbNav" onClick={ slideDown }>	&#8964;</p>
+      <ThumbnailList stylesPhotos={ stylesPhotos } currentThumbs={ currentThumbs } makeActive={ makeActive } activeThumb={ activeThumb }/>
+      <div id="carouselExampleIndicators" className="carousel slide" data-wrap="false" data-ride="false" data-interval="false">
+        <div className="carousel-inner">
+          {photoArr.map((imgObj, index) => {
+            return (<div className={index === 0 ? 'carousel-item active' : 'carousel-item'} data-main-ind={ index } key={ index }>
+              <img src={imgObj.url} className="d-block w-100 main-car img-fluid" alt="..." onClick={ imageExpand }></img>
+            </div>);
```

Comment:
I think this line might read a little easier if it was structured like:
return(
  <div ...
    <img...
  </div>
)

Reply:
But with proper spacing, since github won't let me do it for some reason.

Reply:
Done!